crates/rooch-rpc-server/src/service/aggregate_service.rs

```diff
@@ -196,25 +205,43 @@ impl AggregateService {
             .await?;
 
         assert!(
-            transactions.len() == sequence_infos.len()
+            transactions.len() >= sequence_infos.len()
```

Comment:
The assert will panic the server. It should change to debug_assert, just for debug binary.

Reply:
Or raise an error?

Reply:
change to debug_assert